spell-check/action.yaml

```diff
@@ -13,8 +13,8 @@ inputs:
     description: ""
     required: false
     default: |
-      https://github.com/tier4/autoware-spell-check-dict
-      https://github.com/tier4/cspell-dicts
+      https://github.com/autowarefoundation/autoware-spell-check-dict
+      https://github.com/autowarefoundation/cspell-dicts
```

Comment:
```suggestion
```

I've removed this second `cspell-dicts` repository. We can keep using that one from the TIER IV repositories. 👍